src/main/resources/templates/index.html

```diff
@@ -3,59 +3,18 @@
 <head>
     <meta charset="UTF-8">
     <title>Market Data</title>
-    <style>
-        body{font-family: Consolas,serif;}
-        .panel{ width: 400px; height: 400px; border: solid black 1px}
-    </style>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
 </head>
+<body>
 <h1>Market Data API</h1>
-<h2 id="symbol"></h2>
-<body >
+<h2 id="symbol"> </h2> <!-- I had a misunderstanding about this. I see that your JS is using innerHTML to work with this element. This is fine. It was throwing an error before because your JavaScript was not being called upon as it was located in the wrong place.-->
+<img src="placeholder.png" alt ="Placeholder image of a chart"> <!--If you want to add an image, do it like this using an alt tag. Using empty headers is bad for accessibility.-->
```

Comment:
This was a slight misunderstanding. When I first opened index.html, it was throwing an error due to this h2. The JavaScript works in tandem with this h2. I thought it was supposed to be displaying an image, but it seems you guys are using it for innerHTML stuff. It's working properly now that your JavaScript is being called correctly. This could potentially pose an issue for screen readers.

Reply:
Big no-no: do not put conversation in source code comments.  That's clutter.  Put the conversation here, in the Files Changed section of the pull request, where you can have a dialog with other developers.